resources/assets/js/components/HRPortal/JobIndexPage.tsx

```diff
@@ -0,0 +1,213 @@
+            />
+          </p>
+          <h2 data-c-font-size="h2" data-c-margin="top(triple) bottom(normal)">
+            <FormattedMessage
+              id="hrPortal.jobPageIndex.jobActionsHeader"
+              description="Header for my job actions section."
+              defaultMessage="My Job Actions"
+            />
+          </h2>
+          <p data-c-margin="bottom(double)">
+            <FormattedMessage
+              id="hrPortal.jobPageIndex.jobActionsMessage"
+              description="Message before list of users job actions."
+              defaultMessage="This is a list of all job actions you are currently participating
+                in. Looking for an older job? Check the 'My Completed Job Actions'
+                section below your active jobs."
+            />
+          </p>
+
+          {/* Users Job Actions List */}
+          {/* Not Completed */}
+          {(notCompletedJobActions.length !== 0 &&
+            notCompletedJobActions.map(
+              (jobAction): React.ReactElement => {
+                return <JobCard {...jobAction} />;
+              },
+            )) || (
+            <p>
+              <FormattedMessage
+                id="hrPortal.jobPageIndex.jobActionsEmpty"
+                description="Message displayed if the jobs actions list is empty."
+                defaultMessage="Claim a job below!"
+              />
+            </p>
+          )}
+
+          {/* Completed */}
+          <CompletedJobsAccordion completedJobActions={completedJobActions} />
+
+          <hr data-c-margin="tb(triple)" data-c-hr="gray" />
+          <h2 data-c-font-size="h2" data-c-margin="top(triple) bottom(double)">
+            <FormattedMessage
+              id="hrPortal.jobPageIndex.preDepartmentName"
+              description="Message before department name."
+              defaultMessage="All Jobs in"
+            />
+            {` ${departmentName}`}
+          </h2>
+          <p data-c-margin="bottom(double)">
+            <FormattedMessage
+              id="hrPortal.jobPageIndex.unclaimedJobsMessage"
+              description="Message before list of unclaimed jobs."
+              defaultMessage="This is the list of all active job actions in your department. From
+                here you can 'claim' a job, which will move it into your jobs list
+                above and allow you to begin working with the hiring manager on
+                finding the best talent possible. If you claim a job by accident, no
+                fear, for you can click into the job summary and remove yourself
+                using the 'Remove Myself From This Job' button."
+            />
+          </p>
+
+          {/* Unclaimed Jobs List */}
+          <section data-c-grid="gutter">
+            {unclaimedJobs.length !== 0 &&
+              unclaimedJobs.map(
+                (unclaimedJob): React.ReactElement => {
+                  return <UnclaimedJobCard {...unclaimedJob} />;
```

Comment:
UnclaimedJobCard should also receive a url. Right now the component title is always a url with blank href.